lib/pages/GuitarTuningPage.dart

```diff
@@ -8,21 +11,31 @@ class GuitarTuningPage extends StatefulWidget {
 
 class _GuitarTuningPageState extends State<GuitarTuningPage> {
   int selected = 0; // zero as none of them selected
-
+  final player = AssetsAudioPlayer();
+   List<String> stringSounds=["assets/sounds/E4_330.wav","assets/sounds/B_247.wav","assets/sounds/G_196.wav","assets/sounds/D_147.wav","assets/sounds/A_110.wav","assets/sounds/E2_82.wav"];
   @override
   void initState() {
     super.initState();
     selected = 0;
   }
 
+  void playsound(int stringNumber){
+     player.open(Audio(stringSounds[stringNumber-1]),
+     autoStart: true,     
+    );  
+    
+  }
+
   onSelected(int stringNumber) {
     if (selected == stringNumber) {
       setState(() {
         selected = 0;
+        playsound(stringNumber);
       });
     } else {
       setState(() {
         selected = stringNumber;
+        playsound(stringNumber);
       });
     }
     print("Selected String: " + selected.toString());
```

Comment:
if selected = stringNbrr don't play the sound, and playskund otherwise. And dont call playsound function inside setState.